example/example.html

```diff
@@ -4,35 +4,60 @@
     <meta charset="UTF-8">
     <title>Example</title>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto:300,500,700" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" href="../build/build.css"/>
+    <link rel="stylesheet" href="../index.css"/>
+    <link rel="stylesheet" href="../node_modules/@nib-styles/v2-icons/dist/compiled.css"/>
```

Comment:
Might be easier to setup an `example.scss` compiled with `sass-composer` setup [here](https://github.com/nib-components/react-form/tree/master/example) and [here](https://github.com/nib-components/react-form/blob/master/package.json#L42)